### PR TITLE
Database docker setup

### PR DESCRIPTION
**Pull request by jesp209i**

### CODE REVIEW

**Comment by jesp209i** [comment]
Migrations working on my machine :)

File: src/MakersOfDenmark.Domain/Models/Address.cs [hunk 1]
```diff
@@ -10,6 +10,10 @@ public class Address : ValueObject
         public string Country { get; private set; }
         public string ZipCode { get; private set; }
 
+        public Address()
```
Comment (mnabe) [inline src/MakersOfDenmark.Domain/Models/Address.cs:13]: Lad os lave den private

File: tests/MakersOfDenmark.WebAPI.Tests/GetAllMakerSpacesRequestHandlerTests.cs [hunk 2]
```diff
@@ -31,18 +32,19 @@ public GetAllMakerSpacesRequestHandlerTests()
...
         {
             //Arrange
-            var makerSpaces = _fixture.CreateMany<MakerSpace>();
+            var makerSpaces = _fixture.Build<MakerSpace>().Without(x => x.Tools).CreateMany();
 
             using var dbContext = new MODContext(_options);
             _dbContext.AddRange(makerSpaces);
+            await _dbContext.SaveChangesAsync();
 
             var sut = new GetAllMakerSpacesRequestHandler(dbContext);
```
Comment (mnabe) [inline tests/MakersOfDenmark.WebAPI.Tests/GetAllMakerSpacesRequestHandlerTests.cs:41]: Er enig i at "sut" skal ændres

File: src/MakersOfDenmark.Infrastructure/Persistence/EntityConfigurations/MakerSpaceConfiguration.cs [hunk 3]
```diff
@@ -0,0 +1,17 @@
...
+using System.Collections.Generic;
+using System.Text;
+
+namespace MakersOfDenmark.Infrastructure.Persistence.EntityConfigurations
+{
+    public class MakerSpaceConfiguration : IEntityTypeConfiguration<MakerSpace>
+    {
+        public void Configure(EntityTypeBuilder<MakerSpace> builder)
+        {
+            builder.HasKey(x => x.Id);
```
Comment (mnabe) [inline src/MakersOfDenmark.Infrastructure/Persistence/EntityConfigurations/MakerSpaceConfiguration.cs:14]: Vi mangler stadig at resolve dette: https://github.com/julian-code/makersofdenmark/pull/7#discussion_r502948074

**Review by mnabe — APPROVED** [review]
+1/Godkendt, men vi mangler stadig at resolve https://github.com/julian-code/makersofdenmark/pull/7#discussion_r502948074

**Pull request merged**